src/Jackett/Indexers/nCore.cs

```diff
@@ -0,0 +1,185 @@
+﻿using CsQuery;
+using Jackett.Models;
+using Jackett.Services;
+using Jackett.Utils;
+using Jackett.Utils.Clients;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NLog;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace Jackett.Indexers
+{
+    public class nCore : BaseIndexer, IIndexer
+    {
+        private readonly string SearchUrl = "https://ncore.cc/torrents.php";
+        private static string LoginUrl = "https://ncore.cc/login.php";
+        private readonly string LoggedInUrl = "https://ncore.cc/index.php";
+        private string cookieHeader = "";
+
+        private readonly string enSearch = "torrents.php?oldal=1&tipus=kivalasztottak_kozott&kivalasztott_tipus=xvidser,dvdser,hdser&mire={0}&miben=name";
+        private readonly string hunSearch = "torrents.php?oldal=1&tipus=kivalasztottak_kozott&kivalasztott_tipus=xvidser_hun,dvdser_hun,hdser_hun,mire={0}&miben=name";
+        private readonly string enHunSearch = "torrents.php?oldal=1&tipus=kivalasztottak_kozott&kivalasztott_tipus=xvidser_hun,xvidser,dvdser_hun,dvdser,hdser_hun,hdser&mire={0}&miben=name";
+
+        private IWebClient webclient;
+
+        public nCore(IIndexerManagerService i, IWebClient wc, Logger l)
+            : base(name: "nCore",
+                description: "A Hungarian private torrent site.",
+                link: new Uri("https://ncore.cc/"),
+                caps: TorznabCapsUtil.CreateDefaultTorznabTVCaps(),
+                manager: i,
+                logger: l)
+        {
+            SearchUrl = SiteLink + enHunSearch;
+
+            if (ConfigData != null)
+            {
+                string hun, eng;
+                Dictionary<string, string>[] configDictionary = JsonConvert.DeserializeObject<Dictionary<string, string>[]>(this.ConfigData["config"].ToString());
+                configDictionary[2].TryGetValue("value", out hun);
+                configDictionary[3].TryGetValue("value", out eng);
+
+                bool isHun = Boolean.Parse(hun);
+                bool isEng = Boolean.Parse(eng);
```

Comment:
Shouldn't isHun and isEng be variables around line 32 which are set by LoadFromSavedConfiguration?

The ConfigData property seems like a bad idea to me, we should either have config loaded using LoadFromSavedConfiguration and applied to any class level variables if needed or saved to disk, adding another copy in memory is duplicate data?